src/config.rs

```diff
@@ -75,68 +75,54 @@ pub struct Config {
-    #[structopt(long = "rrdp-dir", parse(from_os_str), default_value = DEFAULT_RRDP_DIR)]
+    #[structopt(long = "rrdp-dir", value_name = "dir", parse(from_os_str), default_value = DEFAULT_RRDP_DIR)]
     pub rrdp_dir: PathBuf,
 
     /// Delay seconds before writing the notification.xml file
     #[structopt(long = "rrdp-notify-delay", value_name = "seconds", default_value = DEFAULT_RRDP_NOTIFY_DELAY_SECONDS)]
     pub rrdp_notify_delay: u64,
 
-    /// Delay seconds before writing the notification.xml file
-    #[structopt(long = "rrdp-max-deltas", value_name = "number", help = "Optionally set a hard upper limit to the number of deltas included in the notification.xml.")]
+    /// Optional hard upper limit to the number of deltas
+    #[structopt(long = "rrdp-max-deltas", value_name = "number")]
     pub rrdp_max_deltas: Option<usize>,
 
     /// The directory to write Rsync files to
-    #[structopt(long = "rsync-dir", parse(from_os_str), default_value = DEFAULT_RSYNC_DIR)]
+    #[structopt(long = "rsync-dir", value_name = "dir", parse(from_os_str), default_value = DEFAULT_RSYNC_DIR)]
     pub rsync_dir: PathBuf,
 
-    /// Force using directory moves rather than symlinks on unix systems. Mainly added to help
-    /// test the alternative code path.
-    #[structopt(long = "rsync-dir-force-moves")]
+    /// Force using directory moves rather than symlinks on unix systems. Added for unit testing this
+    /// code path, not for giving this bad idea to users! So skip it for structopt.
+    #[structopt(skip)]
     pub rsync_dir_force_moves: bool,
 
     /// Disable writing the rsync files.
     #[structopt(long = "rsync-disable")]
     pub rsync_disable: bool,
 
-    /// The minimum number of seconds that a dangling snapshot or delta must have been published by krill-sync before it can be removed
+    /// Remove unreferenced files and directories older than X seconds
     #[structopt(long = "cleanup-after", value_name = "seconds", default_value = DEFAULT_CLEANUP_SECONDS)]
     pub cleanup_after: i64,
 
-    /// Whether or not localhost connections and self-signed certificates are
-    /// allowed.
+    /// Whether or not localhost connections and self-signed certificates are allowed
     #[structopt(long = "insecure")]
     pub insecure: bool,
 
-    /// The RRDP notification file URI of the Krill instance to sync with
+    /// The public RRDP notification URI
     pub notification_uri: Https,
 
-    /// An optional mapping of the notification file URI path up to the parent directory of
-    /// the notification file itself, to a different base URI / path on disk.
-    ///
-    /// Note: For Krill and RIPE NCC - snapshots and deltas can always be found in subdirectories
-    ///        **under** the base dir of the notification file. However, we may want to revisit this
-    ///        configuration in future to allow for explicit mapping of multiple URI paths.
+    /// Base uri for the notify file on the back-end server. Must end with a slash.
```

Comment:
This looks a little odd in the `--help` text as all of hte other texts are single sentences, while this is two sentences.
```
        --rsync-dir <dir>                The directory to write Rsync files to [default: /var/lib/krill-sync/rsync]
        --source_uri_base <uri>          Base uri for the notify file on the back-end server. Must end with a slash
    -s, --state-dir <dir>                The directory to write state to [default: /var/lib/krill-sync]
```

How about `Slash terminated base uri for the notify file on the back-end server` instead?

Also, perhaps use the term `source` insteasd of `back-end server` as the argument is named `source_uri_base`?

Reply:
good suggestion, pushed